src/material/tree/testing/tree-harness.ts

```diff
@@ -28,4 +28,17 @@ export class MatTreeHarness extends ComponentHarness {
   async getNodes(filter: TreeNodeHarnessFilters = {}): Promise<MatTreeNodeHarness[]> {
     return this.locatorForAll(MatTreeNodeHarness.with(filter))();
   }
+
+  async getTreeStructure(): Promise<string> {
```

Comment:
Is there any motivation for adding this function? Is it to simplify test assertions for a tree and it's state? I'm wondering if it would be better type-wise and assertion-wise to just return a nested array or object instead.